thirdparty/README.md

```diff
@@ -28,3 +28,18 @@ following folders removed:
 -   test
 -   include
 -   third_party
+
+## autodiff
+
+A copy of [autodiff](https://github.com/benjaminhuth/autodiff) revision f4a82c2.This 
+fork differs from the [original library](https://github.com/autodiff/autodiff) by 
+supporting `std::hypot` and some small changes to prevent the library from 
```

Comment:
Hm, would there be a way to upstream these changes? I'm not familiar with the library, but I have a bit of a bad feeling of vendoring a fork that we might not be able to update.

Reply:
Yes, that's probably not the best way. I did this, because I up to now only added support for `std::hypot` in the corner-case needed in ACTS. I think to bring this into autodiff, I would need to generalize this (which should not be a big deal). So the question is if we wait with the PR until then or change this later?